website/blog/authors.yml

```
@@ -1,23 +1,30 @@
 ---
+aleksi:
+  name: Alexey Palazhchenko
+  title: Co-Founder, CTO
+  url: https://www.linkedin.com/in/alexeypalazhchenko/
+  image_url: https://github.com/AlekSi.png
+  page: true
+
 alex:
   name: Alexander Fashakin
   title: FerretDB Team
   url: https://github.com/fashander
   image_url: https://github.com/fashander.png
   page: true
 
-peter:
-  name: Peter Farkas
-  title: Co-Founder, CEO
-  url: https://www.linkedin.com/in/farkasp/
-  image_url: https://github.com/ptrfarkas.png
+alvaro:
+  name: Álvaro Hernández
+  title: Founder and CEO @ OnGres
+  url: https://www.linkedin.com/in/ahachete
+  image_url: https://stackgres.io/img/team/alvaro.jpg
   page: true
 
-aleksi:
-  name: Alexey Palazhchenko
-  title: Co-Founder, CTO
-  url: https://www.linkedin.com/in/alexeypalazhchenko/
-  image_url: https://github.com/AlekSi.png
+artem:
+  name: Artem Ervits
+  title: Solutions Engineer at CockroachDB
+  url: https://www.linkedin.com/in/artemervits
+  image_url: img/blog/artem-ervits.jpeg
```

Looks like the URL was accidentally changed
```suggestion
  image_url: /img/blog/artem-ervits.jpeg
```